### DISCUSSION TITLE
Scaling of Grafana on 4K screens

### DISCUSSION THREAD

**Discussion by alexander-scott** [grafana/grafana#38359] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
We're showing the Build Status Monitor and Grafana on 4K TV screens in our office. While the visualization looks great at 1080p, the font sizes etc stay the same at 4K and thus become really small and impossible to read from a distance. We'd like these scaling features to be added.

**Why is this needed**:
We'd prefer the visualization to scale with the resolution, so that texts, table column widths, etc are always the same in proportion to the screen size to enable us to see it more clearly.

**Comment by torkelo** [comment]
Not an ideal solution but a temporary one is to use browser zoom, it will work as you describe scaling everything.

**Comment by torkelo** [comment]
related to https://github.com/grafana/grafana/issues/1875

**Comment by leosh64** [comment]
Hey @torkelo, thanks for your response. The proposed temporary solution of using browser zoom doesn't necessarily work as we're using embedded devices (TV built-in or minimalist raspberry pi OS) to display the web page. For the raspberry pi, zooming should be possible somehow, for the TV built-in software I highly doubt it.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
